### ISSUE TITLE
SpinButton accessibility issues with internal buttons

### ISSUE THREAD

**Issue by ConorWuertz** [OfficeDev/office-ui-fabric-react#7039]
### Environment Information 
 
- __Package version(s)__: 6.47.0
I also see this issue present in the latest version of https://developer.microsoft.com/en-us/fabric#/components/spinbutton
 
#### Actual behavior:
 
The buttons for the up and down action within the SpinButton component do not have the correct accessibility attributes. They have tabindex = -1 but don't inform screen readers that they should not be actionable. The internal Microsoft Keros tool provides the following information:

Rule | button-name: Buttons must have discernible text  MAS 4.1.2

How to fix | Fix ONE of the following:
Element has a value attribute and the value attribute is empty
Element has no value attribute or the value attribute is empty
Element does not have inner text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element's default semantics were not overridden with role="presentation"
Element's default semantics were not overridden with role="none"
Element has no title attribute or the title attribute is empty

 
#### Expected behavior:
I believe these buttons should override the default semantics of a button element with aria-role="none", or should receive an aria-hidden attribute. Or they could become focusable like a normal button.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? Yes
 
Requested priority: Normal
 
Products/sites affected: Microsoft Teams (Shifts app)

**Comment by kenotron** [comment]
@betrue-final-final - would you mind confirming this is indeed the right behavior?
@ConorWuertz  - would you mind submitting a PR for this?

**Comment by ConorWuertz** [comment]
@kenotron I don't mind, I'll wait till @betrue-final-final weighs in

**Comment by kenotron** [comment]
Okay, I'll wait until ben weighs in.

**Comment by ConorWuertz** [comment]
I took another look at this and noticed that the SpinButton takes aria label props for its increment and decrement buttons. Providing those labels takes care of the Keros error. I'll close this issue out.
